specific/cmo_sale/models/sale.py

```diff
@@ -93,11 +98,11 @@ def _compute_project_number(self):
 
     @api.model
     def _default_covenant(self):
-        convenants = self.env['sale.convenant.description'].search([
+        covenants = self.env['sale.covenant.description'].search([
```

Comment:
What if there are 2 actives (may be by mistake).
I think it is better to use default data for this field (ir.values), it is recommended way to set default.